README.md

```diff
@@ -80,7 +80,7 @@ However this function may return true or false:
 async function test(x) {
   const ref = new WeakRef(x)
   const y = ref.deref();
-  await true;
+  await new Promise(resolve => setTimeout(resolve));
```

Comment:
`await Promise.resolve()` should also force a tick

Reply:
There are multiple notions of "tick". WeakRefs do not become null after `await Promise.resolve()`. This was discussed as an open question in the March 2019 TC39 meeting, and discussed it further in this repository. I put some of my rationale in https://github.com/tc39/proposal-weakrefs/issues/61#issuecomment-483465169 ; @tschneidereit has also expressed concern about potential compatibility issues between implementations if we used microtask timing.

Reply:
ah, ok - since setTimeout isn’t in the language, how else would you do this?

Reply:
There are embedder hooks in the specification, as there are in several places in the JavaScript specification. Please file an issue if you want to discuss your concerns in more detail.

Reply:
Embedded hooks don’t mandate the engine provide a mechanism to JS.

Reply:
I've searched and real all issues and comments, but I can'f find anywhere to this question I've had before: why is it not allowed to let the host run the `ClearKeptObjects` and `CleanupFinalizationGroup` jobs after the current synchronous job?
In that case `await Promise.resolve()` may be sufficient, but it also may not.
The comments in various issues seem to indicate the host MUST wait a full task turn. I just don't understand why the requirement.